focuspoints.lrdevplugin/focus_points/pentax/pentax k-50.txt

```diff
@@ -2,8 +2,6 @@
 -- Submitted by: roguephysicist
 -- 16.3 MP, 4928x3264
 -- AF System: 11 points, 9 cross-type
--- selected points: AFPointSelected
--- in-focus points: AFPointsInFocus
```

Comment:
I actually made a PR to add this feature. It was a simple little change. Would you mind taking a look at it for me? What should be the real height of focus points on the end? 

https://github.com/musselwhizzle/Focus-Points/pull/125/files